### DISCUSSION TITLE
Screensaver not working on lock screen

### DISCUSSION THREAD

**Discussion by DaveMeek** [rocksdanister/lively#2122] [General]
Hello 

I can not get the screen saver to work on the lock screen, I'm just using one of the built in ones but every time I lock the PC it flcikers like the screensaver is trying to start but it just doesn't.

Built in windows  screensavers work fine and the lively screen works when my PC isn't locked.

I followed the instructions here https://github.com/rocksdanister/lively/wiki/Screen-Saver

But can not figure out why its not working

Please could someone assist me?

**Comment by rocksdanister** [comment]
Makes sense, since Lively screensaver .scr file is just calling the Lively Wallpaper program in the background it won't work inside restricted lockscreen area (Windows expects .scr file themselves to be the screensaver, not different program.)

Not sure if this can be solved without having to built a new screensaver only program which is too much work.

**Reply by bolanxno7gan** [reply]
awww the only reason i got lively lol
